src/components/EmailsList.tsx

```diff
@@ -0,0 +1,217 @@
+import * as React from "react";
+import { ComponentConfig, Fields } from "@measured/puck";
+import {
+  EntityField,
+  YextEntityField,
+  resolveYextEntityField,
+  YextEntityFieldSelector,
+  useDocument,
+} from "@yext/visual-editor";
+import { cn } from "../utils/cn";
+import { cva, type VariantProps } from "class-variance-authority";
+import mailIcon from "../assets/mail_outline.svg";
+
+
+const emailListVariants = cva("list-inside text-font-fontSize p-8", {
+  variants: {
+    fontSize: {
+      xs: "text-xs",
+      sm: "text-sm",
+      medium: "text-base",
+      lg: "text-lg",
+      xl: "text-xl",
+      "2xl": "text-2xl",
+      "3xl": "text-3xl",
+      "4xl": "text-4xl",
+      "5xl": "text-5xl",
+    },
+    fontWeight: {
+      default: "font-body-fontWeight",
+      thin: "font-thin",
+      extralight: "font-extralight",
+      light: "font-light",
+      normal: "font-normal",
+      medium: "font-medium",
+      semibold: "font-semibold",
+      bold: "font-bold",
+      extrabold: "font-extrabold",
+      black: "font-black",
+    },
+    color: {
+      default: "text-body-color",
+      primary: "text-primary",
+      secondary: "text-secondary",
+      accent: "text-accent",
+      text: "text-text",
+      background: "text-background",
+      foreground: "text-foreground",
+    },
+    textTransform: {
+      none: "",
+      uppercase: "ve-uppercase",
+      lowercase: "ve-lowercase",
+      capitalize: "ve-capitalize",
+    },
+    padding: {
+      default: "px-4 py-16 md:px-8",
+      none: "p-0",
+      small: "px-4 py-8 md:px-8",
+      large: "px-[200px] py-24 md:px-8",
+    },
+  },
+  defaultVariants: {
+    padding: "none",
+    fontWeight: "default",
+    color: "default",
+    textTransform: "none",
+  },
+});
+
+export interface EmailListProps extends VariantProps<typeof emailListVariants> {
+  list: YextEntityField<string[]>;
+  includeHyperlink: boolean;
+  listLength: number;
+}
+
+const EmailListFields: Fields<EmailListProps> = {
+  list: YextEntityFieldSelector({
+    label: "Entity Field",
+    filter: {
+      types: ["type.string"],
+      allowList: ["emails"],
+      includeListsOnly: true,
+    },
+  }),
+  fontSize: {
+    label: "Font Size",
```

Comment:
add default font size?

Reply:
Originally I had a default, but since default is the medium size it was weird to have default first and not be the smallest.

But we also don't want default to be the smallest size or to have default in the middle of the list.

I think the solution I went with makes the most sense but I'm open to suggestions.

Reply:
Shouldn't be a number input like we're doing for HeaderText and BodyText?

Reply:
For heading text it translates to <h1> <h2> etc

Body text does not have a text size option

Reply:
I think this is ok for now and [SUMO-6627](https://yexttest.atlassian.net/jira/software/c/projects/SUMO/boards/97?selectedIssue=SUMO-6627) should change this to respect the body font size